src/functions.js

```diff
@@ -583,6 +587,9 @@ defineFunction(["\\over", "\\choose"], {
         case "\\choose":
             replaceWith = "\\binom";
             break;
+        case "\\atop":
+            replaceWith = "\\\\atopfrac";
+            break;
```

Comment:
I added a Jasmine test, not everything needs a screenshot test. But ok fine whatevs